pkgs/development/python-modules/deltalake/default.nix

```
@@ -15,29 +16,29 @@
   pytest-cov-stub,
   pytest-mock,
   pandas,
+  deprecated,
   azure-storage-blob,
 }:
 
 buildPythonPackage rec {
   pname = "deltalake";
-  version = "0.25.5";
+  version = "1.1.2";
   format = "pyproject";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-Fz5Lg/z/EPJkdK4RcWHD8r3V9EwwwgRjwktri1IOdlY=";
+    hash = "sha256-s/iWYoh2zARl3M+0DPdur5d8a1URl+jinaMPBFeruEE=";
   };
 
   cargoDeps = rustPlatform.fetchCargoVendor {
     inherit src;
-    hash = "sha256-6SGVKJu01MzZxJv29PZKea+Z2YwAnvzbdDlnA4R6Az0=";
+    hash = "sha256-JYstNjd/KC9xp2h72vkQfin/LXNTXeb0hLpGUiGgRlE=";
   };
 
   env.OPENSSL_NO_VENDOR = 1;
 
   dependencies = [
-    pyarrow
-    pyarrow-hotfix
+    arro3-core
   ];
```

delta-rs also has deprecated in dependencies https://github.com/delta-io/delta-rs/blob/main/python/pyproject.toml#L20. Should we have it here also?
```suggestion
    arro3-core
    deprecated
  ];
```